apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -241,20 +236,20 @@ var TopInstructions = React.createClass({
               />}
             {experiments.isEnabled('resourcesTab') &&
               <div style={styles.helpTabs}>
-                <a
-                  className="uitest-instructionsTab"
+                <Tab
+                  class="uitest-instructionsTab"
                   onClick={this.handleInstructionTabClick}
-                  style={{...styles.tab, ...(!this.state.helpTabSelected && styles.highlighted)}}
-                >
-                  {msg.instructions()}
-                </a>
-                <a
-                  className="uitest-helpTab"
-                  onClick={this.handleHelpTabClick}
-                  style={{...styles.tab, ...(this.state.helpTabSelected && styles.highlighted)}}
-                >
-                  {msg.helpTips()}
-                </a>
+                  style={this.state.helpTabSelected? null : styles.highlighted}
```

Comment:
nit: here and in a few places instead of `foo? a: b` we should have a space, i.e. `foo ? a : b`.

One reason for this is that in ruby it's common to have method names that are something like `foo?`, so if we don't have the space in JS, it starts look like it's a method.